packages/core/src/components/Review/Review.jsx

```diff
@@ -0,0 +1,69 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import ReviewLink from './ReviewLink';
+import classNames from 'classnames';
+
+/*
+`<Review>`
+
+@react-component Review
+
+Style guide: components.review.react
+*/
+
+/**
+ * The `Review` component is used for listing a summary of information entered
+ * by a user. Its content includes a heading, value, and edit link.
+ */
+export class Review extends React.PureComponent {
+  heading() {
+    if (this.props.heading) {
+      return (
+        <h3 className="ds-c-review__heading ds-text ds-u-margin-bottom--0 ds-u-font-weight--bold ds-u-display--inline-block">
+          {this.props.heading}
+        </h3>
+      );
+    }
+  }
+
+  render() {
+    const classes = classNames(
+      'ds-c-review ds-u-border-bottom--2 ds-u-padding-y--2 ds-u-justify-content--between ds-u-display--flex',
+      !this.props.alignTop && 'ds-u-align-items--center'
+    );
+    return (
+      <div className={classes}>
+        <div className="ds-u-margin-right--2">
+          {this.heading()}
+          <div className="ds-c-review__body">{this.props.children}</div>
+        </div>
+        <ReviewLink onClick={this.props.onClick} href={this.props.href}>
+          {this.props.linkText}
+        </ReviewLink>
+      </div>
+    );
+  }
+}
+
+Review.defaultProps = {
+  alignTop: true,
+  linkText: 'Edit'
+};
+
+Review.propTypes = {
```

Comment:
Could we add support for a `className` prop so someone can pass in additional classes to be added to the root element? In your example, `ds-u-margin-top--6` could then be added directly to the `Review` component.